components/webgpu/Cargo.toml

```diff
@@ -15,7 +15,8 @@ embedder_traits = {path = "../embedder_traits"}
 ipc-channel = "0.14"
 log = "0.4"
 malloc_size_of = { path = "../malloc_size_of" }
-serde = "1.0"
+serde = { version = "1.0", features = ["serde_derive"] }
 servo_config = {path = "../config"}
 smallvec = { version = "0.6", features = ["serde"] }
 wgpu-core = { version = "0.1.0", git = "https://github.com/gfx-rs/wgpu", features = ["serde"] }
+wgpu-types = { version = "0.1.0", git = "https://github.com/gfx-rs/wgpu" }
```

Comment:
why are we using version 0.1.0 for wgpu? it should be 0.5

Reply:
You're right. Initially I was a bit worried to do that in a single step, since there are quite a lot of changes. But while working on this I realised doing it in steps would just confuse and complicate things. I'll pause on this PR for a while and work on upgrading the version first, if that's okay with you.